### PR TITLE
Director/surveys

### LINKED ISSUE #39: Director: Survey results view
1.8

### PR DESCRIPTION
**Pull request by swopl**
Need better names from some parts. Maybe DTO instead of returning tuples?
Resolves #39

### CODE REVIEW

File: HospitalSystem.ConsoleUI/Director/Surveys/DoctorSurveyUI.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    private List<DoctorSurvey> _loadedSurveys;
+
+    public DoctorSurveyUI(Hospital hospital) : base(hospital)
+    {
+        _loadedSurveys = new();
+    }
+
+    private void RefreshSurveys()
+    {
+        _loadedSurveys = _hospital.SurveyService.GetAllDoctor().ToList();
```
Comment (KatarinaM-SV6) [inline HospitalSystem.ConsoleUI/Director/Surveys/DoctorSurveyUI.cs:17]: Should be called GetAllDoctors? plural not singular

**Review by KatarinaM-SV6 — APPROVED** [review]
I like it.

File: HospitalSystem.ConsoleUI/Director/Surveys/DoctorSurveyUI.cs (hunk 1 above)
Comment (swopl) [reply HospitalSystem.ConsoleUI/Director/Surveys/DoctorSurveyUI.cs:17]: This does not get Doctors. It gets DoctorSurveys

Comment (KatarinaM-SV6) [reply HospitalSystem.ConsoleUI/Director/Surveys/DoctorSurveyUI.cs:17]: Okay!

**Pull request merged**